universal-login-ops/src/dev/deployWalletMaster.js

```diff
@@ -1,8 +1,11 @@
-import WalletMaster from '@universal-login/contracts/build/WalletMaster';
+import WalletMaster from '@universal-login/contracts/build/WalletMaster.json';
 import {deployContract} from 'ethereum-waffle';
+import {utils} from 'ethers';
 
 export default async function deployWalletMaster(deployWallet) {
   const {address} = await deployContract(deployWallet, WalletMaster);
   console.log(`WalletMaster address: ${address}`);
-  return address;
+  const masterContractHash = utils.keccak256(`0x${WalletMaster.evm.deployedBytecode.object}`);
```

Comment:
To consider - extract helpers: 
* `getContractDeplyedBaytecode()`
* `getContractHash()`

Reply:
✔️